site/reference/fn/source/README.md

```diff
@@ -1,71 +1,64 @@
 ---
-title: "Source"
+title: "`source`"
 linkTitle: "source"
 type: docs
 description: >
-   Specify a directory as an input source package
+   Source resources from a local directory
 ---
 
 <!--mdtogo:Short
-    Specify a directory as an input source package
+    Source resources from a local directory
 -->
 
-Implements a [source function] by reading configuration and writing to STDOUT.
+`source` reads resources from a local directory and writes them in [Function Specification]
+wire format to `stdout`. The output of the `source` can be pipe'd to commands
+such as `kpt fn eval` that accepts [Function Specification] wire format. `source`
+is useful for chaining functions using Unix pipe. For more details, refer to
+[Chaining functions] and [Function Specification].
 
-### Examples
-
-{{% hide %}}
+### Synopsis
 
-<!-- @makeWorkplace @verifyExamples-->
-```
-# Set up workspace for the test.
-TEST_HOME=$(mktemp -d)
-cd $TEST_HOME
-```
+<!--mdtogo:Long-->
 
-<!-- @fetchPackage @verifyExamples-->
 ```shell
-export SRC_REPO=https://github.com/GoogleContainerTools/kpt.git
-kpt pkg get $SRC_REPO/package-examples/helloworld-set@next DIR/
+kpt fn source [DIR] [flags]
 ```
 
-{{% /hide %}}
+#### Args
 
-<!--mdtogo:Examples-->
-
-<!-- @fnSource @verifyExamples-->
 ```shell
-# print to stdout configuration from DIR/ formatted as an input source
-kpt fn source DIR/
+DIR:
+  Path to the local directory containing resources. Defaults to the current
+  working directory.
 ```
 
+#### Flags
+
 ```shell
```

Comment:
nit: other commands don't use `shell` directive, so there's little bit of inconsistency in rendering

Reply:
Thanks for point it out. Cleaned it up.